app/lib/services/restApi.ts

```diff
@@ -347,9 +348,9 @@ export const getTeamListRoom = ({
 	return sdk.get('teams.listRooms', params);
 };
 
-export const closeLivechat = (rid: string, comment?: string) =>
+export const closeLivechat = (rid: string, comment?: string, tags?: string[]) =>
 	// RC 0.29.0
-	sdk.methodCallWrapper('livechat:closeRoom', rid, comment, { clientAction: true });
+	sdk.methodCallWrapper('livechat:closeRoom', rid, comment, { clientAction: true, tags });
```

Comment:
Are you sure old versions of the server are accepting this extra argument?
I'm not talking about disabling the feature, but backwards compatibility.

Reply:
I don't have this answer right now, but I will search for it. At least, in which version of the server the tags were introduced

Reply:
The tags were introduced here: https://github.com/RocketChat/Rocket.Chat/pull/17254

2 years ago!

Instead of creating a condition for which version of the server, I decided to send the extra argument only when there are tags selected.